src/components/Button.js

```diff
@@ -44,79 +71,39 @@ const Base = styled.button`
   }
 `
 
-const emphasized = withProps(props => {
-  if (props.emphasized) {
-    return {
-      px: 5
-    }
-  }
-})
-
-const setDefaultProps = withProps(props => {
-  // set defaults
-  // always allow override with provided props
-  const color = props.color || props.theme.colors.text.main
-  return Object.assign(
-    {
-      bg: 'transparent',
-      color: color,
-      px: 16,
-      pb: 0,
-      border: '1px solid',
-      borderColor: lighten(color),
-      active: {
-        color: '#fff',
-        bg: color
-      }
-    },
-    props
-  )
-})
-
-const getType = withProps(props => {
-  // get primary, tertiary, secondary and set as props.type
-  const type = Object.keys(props).find(b =>
-    Object.keys(props.theme.colors).find(k => k === b)
-  )
-  props.type = type
-  return props
-})
+const Outline = Button.extend`
+  color: ${props =>
+    getColor(props, 'color', 'main') || props.theme.colors.text.main};
+  background: ${props => props.color || 'none'};
+  border: 1px solid;
+`
 
-const setTypeProps = withProps(({ type, theme }) => {
-  // set type colors
-  if (!type) return
+const Underline = Outline.extend`
+  border: 0;
+  border-radius: 0;
+  border-bottom: 1px solid;
 
-  return {
-    color: '#fff',
-    bg: get(theme.colors[type], 'main'),
-    active: {
-      color: '#fff',
-      bg: get(theme.colors[type], 'dark')
-    }
+  &:hover,
+  &:focus,
+  &:active {
+    background: none;
```

Comment:
Just a note, in the billing page of the dashboard the underline buttons get filled on hover/focus/active, looking similar with the hovered outlined buttons. Never had a spec on this and neither have a strong opinion though.

Reply:
Hmm, yea I'm not sure either? I was basing it off the designs for the marketing site pricing page. Great that we are catching these inconsistencies :) @konmouz any opinions? If not we can create an issue and change the button when we have a design for it.

Reply:
@craig-mulligan @thgreasi, at the moment @taahirisaacs is preparing a new styleguide version that will be shared soon. After that, we can review what elements are missing and populate the guide.

Reply:
Okay thanks @konmouz, I'll create an issue so we can revisit it once we have designs.